tests/comment.test.js

```diff
@@ -2,8 +2,8 @@ const request = require('supertest');
 
 const dbHandler = require('./db.handler');
 const storyDao = require('../dao/story.dao.server');
-const storyModel = require('../models/story.model.server');
```

Comment:
why does this need to be removed?

Reply:
Because of this package never used.

Reply:
it was used to create test stories, type safety.